cypress/integration/desktop/desktopVisitorCanNavigateBetweenSPAViews.feature.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-undef */
 describe("visitor can navigate between views", () => {
   beforeEach(() => {
-    cy.viewport("macbook-15");
+    
```

Comment:
why remove?

Reply:
had problems with it yesterday, but now reinstated back and it works well... don't know what happened, guess I was tired. It is back now